specification/dashboard/resource-manager/readme.md

```diff
@@ -44,7 +44,7 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-net
+  - repo: azure-sdk-for-net-track2
```

Comment:
I don't see any obvious service directory for this in Azure/azure-sdk-for-net. Is there even a track 2?

Reply:
It's called grafana in azure-sdk-for-net as you can see the readme is used in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/grafana/Azure.ResourceManager.Grafana/src/autorest.md